### DISCUSSION TITLE
Extend the title (structure) in settings

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#366] [Q&A] [answered]
After a search in the settings, it is not easy to recognize where the parameter is located in the structure. The sub-items "Advanced" and "Appearance" in particular appear several times. Wouldn't it make sense to expand the titles so that the structure is recognizable? Example:
<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/2ab5b66b-698e-44fb-89c0-95d57ae11c82"/>     <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/3d910ced-14e9-425e-ae7d-816ea20896ee"/>    <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/bdbcfe59-c07e-42db-be50-fbee8f054c2b"/>

**Comment by devemux86** [comment]
@WolfgangZ42 thanks for the suggestion.

These are automatically labeled by Android. I will need to check if it is possible to change them.

**Comment by devemux86** [comment]
Settings title with structure is available in the latest:
- https://github.com/devemux86/cruiser/discussions/451

**Comment by devemux86** [comment, marked as answer]
Settings title with structure is available in [Cruiser 3.0.41](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
